src/action-ros-ci.ts

```diff
@@ -17,6 +17,7 @@ const validROS2Distros: string[] = [
 	"foxy",
 	"galactic",
 	"humble",
+	"iron",
```

Comment:
you'll also need to re-generate the `dist/index.js` file. See: https://github.com/ros-tooling/action-ros-ci/blob/master/DEVELOPING.md

You can also just modify it manually, since you're only adding a single line.

Reply:
Oh thanks, sorry I saw the generated code checked in and assumed it would get generated post merge, didn’t read the dev doc 🙈